### DISCUSSION TITLE
Is there ever going to be an official release again?

### DISCUSSION THREAD

**Discussion by EmmanuelGoldstein** [wasp-os/wasp-os#448] [General]
So I am just a normal user and I flashed the latest official release onto my watch and realized it's hopelessly outdated and doesn't contain any of the more recent apps or functionality. And I tried to get the CS releases to run, but they all have the problem that I can't turn to page 2 in the software app, making it impossible for me activate most of the apps. Anyway, is there ever going to be a reliable official release again that can actually be used by normal users like me?

**Comment by thiswillbeyourgithub** [comment]
AFAIK the CS is supposed to be stable. If not you should open an issue and this will get fixed eventually!

**Comment by daniel-thompson** [comment]
The current maintainer (me) is looking for new blood to come into the project. It's one of the reasons we moved from my own github account to the wasp-os one!

Right now I'm more or less retired although I do still try to clear things off the pull-request queue when I get the chance. Without additional volunteers I'm not sure there will be a new release any time soon (in the mean time I wouldn't object to proposed changes to the docs to increase the prominence of the CI builds.
